index.js

```diff
@@ -29,36 +32,36 @@ updateNotifier({pkg: cli.pkg}).notify()
 
 const run = () => {
   const today = new Date()
-  const day = today.getUTCDate()
+  const day = !cli.flags.m ? today.getUTCDate() : undefined
   const month = today.getUTCMonth() + 1
+  const country = cli.flags.br ? 'br' : 'us'
+  const country_emoji = cli.flags.br ? '🇧🇷' : '🇺🇸'
```

Comment:
Don't do unrelated changes. Please revert it. You can create another pull request or issue for that. If you do, I recommend using an array instead since we can eventually support more countries.